src/dictionary.ts

```diff
@@ -1949,6 +1997,12 @@ export const en_dk : Lexeme[] = [
     translations: ["validering, -en, -er, -erne"],
     keywords: []
   },
+  {
+    word: "verification",
+    type: "sb.",
+    translations: ["bevisførelse, -n, -r, -rne"],
```

Comment:
"verifikation, -en, -er, -erne" ?

Reply:
This one came from the SML Wikipedia entry: https://da.wikipedia.org/wiki/Standard_ML

> Sproget er populært til udvikling af compilere, forskning omkring programmeringssprog og udvikling af systemer til automatisk bevisførelse. 

I guess bevisførelse is more like proof-generation?

Reply:
Yes, I would translate that back to "writing a prove" or "the act of proving".